client/components/Pedalboards/Pedalboards.js

```diff
@@ -2,13 +2,16 @@ import React from 'react'
 import PedalBoard from './Pedalboard/Pedalboard'
 import { Container, Card } from 'semantic-ui-react'
 
-const pedalBoards = ({ pedalData }) => {
+const pedalBoards = ({ handleClick, pedalData }) => {
   return (
     <Container>
       <Card.Group itemsPerRow={4} className="ui center aligned grid">
         {pedalData.map((elem, index) => {
           return (
             <PedalBoard
+              key={index}
+              id={index}
```

Comment:
Don't your objects have a real id?

Reply:
@thebearingedge yes, but it's a UUID. I'm passing the index as ID value back into my handler so that I can easily locate the one I want.  Should I do it a different way?

Reply:
You can use `Array.prototype.find` to compare the `id` of each object with the given `id` right?